.gitignore

```diff
@@ -28,7 +28,7 @@ Gemfile*.lock
 
 # Berkshelf
 Berksfile.lock
-cookbooks/
+/cookbooks/
```

Comment:
why do we still want to keep `<root>/cookbooks`?

Reply:
I assumed this was the original intent of the line, but I'm not sure if we need it. Just keeping changes to the minimum.

Reply:
I'll delete it and if we see we need it we can add it again.